lib/workers/branch/index.ts

```diff
@@ -342,97 +342,115 @@ export async function processBranch(
       global.trustLevel === 'high' &&
       is.nonEmptyArray(config.allowedPostUpgradeCommands)
     ) {
-      logger.debug(
-        {
-          tasks: config.postUpgradeTasks,
-          allowedCommands: config.allowedPostUpgradeCommands,
-        },
-        'Checking for post-upgrade tasks'
-      );
-      const commands = config.postUpgradeTasks.commands || [];
-      const fileFilters = config.postUpgradeTasks.fileFilters || [];
+      for (const upgrade of config.upgrades) {
+        addMeta({ dep: upgrade.depName });
```

Comment:
```suggestion
```

It's not necessary to include this if you're adding `dep` in the `logger.trace` command anyway.

Reply:
That's for `logger.debug` down below. (Line 380, 392, 398, 436)

Reply:
In that case, can you remove the explicit `dep` from any `logger` commands, as there's no reason to duplicate them